<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Arrowgame</class>
 <widget class="QMainWindow" name="Arrowgame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>942</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Гра Стрілки</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="to_solve">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>240</y>
      <width>151</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Розв'язати програмно</string>
    </property>
   </widget>
   <widget class="QTableView" name="tableView">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>170</y>
      <width>251</width>
      <height>211</height>
     </rect>
    </property>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>42</number>
    </attribute>
    <attribute name="horizontalHeaderMinimumSectionSize">
     <number>40</number>
    </attribute>
    <attribute name="verticalHeaderDefaultSectionSize">
     <number>40</number>
    </attribute>
    <attribute name="verticalHeaderMinimumSectionSize">
     <number>40</number>
    </attribute>
   </widget>
   <widget class="QSpinBox" name="spinBox">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>40</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>2</number>
    </property>
    <property name="maximum">
     <number>8</number>
    </property>
    <property name="value">
     <number>2</number>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>40</y>
      <width>171</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Введіть розмір головоломки</string>
    </property>
   </widget>
   <widget class="QTableView" name="downarrows">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>380</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>42</number>
    </attribute>
    <attribute name="horizontalHeaderMinimumSectionSize">
     <number>40</number>
    </attribute>
    <attribute name="verticalHeaderDefaultSectionSize">
     <number>31</number>
    </attribute>
    <attribute name="verticalHeaderMinimumSectionSize">
     <number>31</number>
    </attribute>
   </widget>
   <widget class="QTableView" name="leftarrows">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>171</y>
      <width>31</width>
      <height>211</height>
     </rect>
    </property>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>49</number>
    </attribute>
    <attribute name="horizontalHeaderMinimumSectionSize">
     <number>31</number>
    </attribute>
    <attribute name="verticalHeaderDefaultSectionSize">
     <number>40</number>
    </attribute>
    <attribute name="verticalHeaderMinimumSectionSize">
     <number>40</number>
    </attribute>
   </widget>
   <widget class="QTableView" name="rightarrows">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>170</y>
      <width>31</width>
      <height>211</height>
     </rect>
    </property>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>49</number>
    </attribute>
    <attribute name="horizontalHeaderMinimumSectionSize">
     <number>31</number>
    </attribute>
    <attribute name="verticalHeaderDefaultSectionSize">
     <number>40</number>
    </attribute>
    <attribute name="verticalHeaderMinimumSectionSize">
     <number>40</number>
    </attribute>
   </widget>
   <widget class="QPushButton" name="to_gen">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>40</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Згенерувати</string>
    </property>
   </widget>
   <widget class="QPushButton" name="to_enter">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>40</y>
      <width>151</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Ввести самостійно</string>
    </property>
   </widget>
   <widget class="QPushButton" name="to_check">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>160</y>
      <width>151</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string> Перевірити ввід</string>
    </property>
   </widget>
   <widget class="QPushButton" name="to_play">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>200</y>
      <width>151</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Розв'язати самостійно</string>
    </property>
   </widget>
   <widget class="QPushButton" name="b1">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>234</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="b2">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>230</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="b3">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>230</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="to_confirm">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>360</y>
      <width>191</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Перевірити розв'язок</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>510</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="to_write">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>510</y>
      <width>171</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Записати розв'язок у файл</string>
    </property>
   </widget>
   <widget class="QPushButton" name="to_hint">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>290</y>
      <width>151</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Використати підказку</string>
    </property>
   </widget>
   <widget class="QLabel" name="lab">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>490</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Ім'я файлу</string>
    </property>
   </widget>
   <widget class="QTableView" name="uparrows">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>140</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <property name="showGrid">
     <bool>true</bool>
    </property>
    <property name="sortingEnabled">
     <bool>false</bool>
    </property>
    <property name="cornerButtonEnabled">
     <bool>true</bool>
    </property>
    <attribute name="horizontalHeaderMinimumSectionSize">
     <number>40</number>
    </attribute>
    <attribute name="verticalHeaderDefaultSectionSize">
     <number>31</number>
    </attribute>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
